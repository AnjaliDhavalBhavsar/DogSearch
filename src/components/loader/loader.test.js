import renderer from "react-test-renderer";
import { Loader } from ".";

test("Loader testing", () => {
  const element = renderer.create(<Loader />).toJSON();
  expect(element).toMatchSnapshot();
});
