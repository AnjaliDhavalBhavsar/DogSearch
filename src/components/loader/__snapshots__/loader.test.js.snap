// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader testing 1`] = `
<div
  className=""
>
  <div
    className="loader"
  />
   
</div>
`;
